src/pkcs1v15.rs

```diff
@@ -1,12 +1,17 @@
 use alloc::vec;
 use alloc::vec::Vec;
 use rand_core::{CryptoRng, RngCore};
+use signature::{RandomizedSigner, Signer, Verifier};
 use subtle::{Choice, ConditionallySelectable, ConstantTimeEq};
 use zeroize::Zeroizing;
 
+use crate::dummy_rng::DummyRng;
 use crate::errors::{Error, Result};
 use crate::hash::Hash;
 use crate::key::{self, PrivateKey, PublicKey};
+use crate::{PublicKeyParts, RsaPrivateKey, RsaPublicKey};
+
+pub use crate::Signature;
```

Comment:
I'd suggest giving `pkcs1v15` and `pss` different `Signature` newtypes in order to make them type safe so they don't get confused for one another

Reply:
Ack. Initially I had two different types, which had 100% of common code. I'll change this back.